client/state/data-layer/offline-dev/plans-request.js

```diff
@@ -0,0 +1,526 @@
+			AUD: {
+				original: 129,
+				price_c: 161.84,
+				price_d: 129
+			},
+			CAD: {
+				original: 129,
+				price_c: 161.84,
+				price_d: 129
+			},
+			EUR: {
+				original: 99,
+				price_c: 119.88,
+				price_d: 99
+			},
+			GBP: {
+				original: 85,
+				price_c: 89.91,
+				price_d: 85
+			},
+			JPY: {
+				original: 11800,
+				price_c: 14985,
+				price_d: 11800
+			},
+			USD: {
+				original: 99,
+				price_c: 119.88,
+				price_d: 99
+			}
+		},
+		prices: {
+			AUD: 129,
+			CAD: 129,
+			EUR: 99,
+			GBP: 85,
+			JPY: 11800,
+			NZD: 139,
+			USD: 99
+		},
+		product_id: 1003,
+		product_name: 'WordPress.com Premium',
+		product_name_short: 'Premium',
+		product_slug: 'value_bundle',
+		product_type: 'bundle',
+		raw_price: 16998,
+		store: 0,
+		support_document: 'bundles',
+		tagline: 'Supercharge your site',
+		width: 500
+	},
+	{
+		android_sku: 'sub_test_plan_business_001',
+		apple_sku: 'com.wordpress.test.business.subscription.1year',
+		available: 'yes',
+		bill_period: 365,
+		bill_period_label: 'per year',
+		bundle_product_ids: [
+			12,
+			45,
+			15,
+			48,
+			50,
+			49,
+			5,
+			6,
+			46,
+			54,
+			56,
+			57,
+			58,
+			59,
+			60,
+			61,
+			62,
+			63,
+			64,
+			65,
+			66,
+			67,
+			68,
+			72,
+			73,
+			74,
+			75,
+			16
+		],
+		capability: 'manage_options',
+		cost: 299,
+		currency_code: 'USD',
+		description: 'Everything included with Premium, as well as live chat support, and unlimited access to our premium themes.',
+		features_highlight: [
+			{
+				items: [
+					'premium-themes',
+					'space',
+					'support'
+				]
+			},
+			{
+				items: [
+					'custom-design',
+					'videopress',
+					'no-adverts/no-adverts.php',
+					'custom-domain'
+				],
+				title: 'Includes WordPress.com Premium features:'
+			},
+			{
+				items: [
+					'free-blog'
+				],
+				title: 'Included with all plans:'
+			}
+		],
+		formatted_price: '$299',
+		height: 435,
+		icon: 'https://s0.wordpress.com/i/store/plan-business.png',
+		icon_active: 'https://s0.wordpress.com/i/store/plan-business-active.png',
+		price: '$299',
+		price_variants: {
+			AUD: {
+				original: 399,
+				price_c: 404.84,
+				price_d: 323.84
+			},
+			CAD: {
+				original: 389,
+				price_c: 404.84,
+				price_d: 323.84
+			},
+			EUR: {
+				original: 299,
+				price_c: 299.88,
+				price_d: 239.88
+			},
+			GBP: {
+				original: 250,
+				price_c: 224.91,
+				price_d: 179.91
+			},
+			JPY: {
+				original: 35800,
+				price_c: 37485,
+				price_d: 29985
+			},
+			USD: {
+				original: 299,
+				price_c: 299.88,
+				price_d: 239.88
+			}
+		},
+		prices: {
+			AUD: 399,
+			CAD: 389,
+			EUR: 299,
+			GBP: 250,
+			JPY: 35800,
+			NZD: 399,
+			USD: 299
+		},
+		product_id: 1008,
+		product_name: 'WordPress.com Business',
+		product_name_short: 'Business',
+		product_slug: 'business-bundle',
+		product_type: 'bundle',
+		raw_price: 299,
+		tagline: 'Take it to the next level',
+		width: 500
+	}
+];
```

Comment:
It may be worth separating out this data into another file so that we can reuse it. I imagine having a library of data available for the different types of data objects.

```js
import plansData from './data/plans'

…
```